File: 2 Semestre/Sistemas Embarcados (C)/Aula 4/Exercicio 3.c
#include <stdio.h>

int main(){
    float p_combustivel, desempenho, d_cidades, litros, valor;
    
    printf("Digite a distancia entre a cidade de origem e a cidade de destino: ");
    scanf("%f",&d_cidades);

    printf("Digite o preco do combustivel: ");
    scanf("%f",&p_combustivel);

    printf("Digite o desempenho do carro por litro de combustivel (km/l): ");
    scanf("%f",&desempenho);

    litros = d_cidades / desempenho;
    valor =  litros * p_combustivel;

    printf("Voce precisara de %0.2f litros que custaram R$ %0.2f",litros,valor); }

File: 2 Semestre/Programacao de Computadores/Aula 7/Exercicio 3.py

def mdc(a,b):
    if b <= a and a % b == 0: return b
    elif a < b: return mdc(b,a)
    return mdc(b, a % b)

a = int(input("Digite o valor de A: "))
b = int(input("Digite o valor de B: "))
print(f"MDC = {mdc(a,b)}")